forum/communications/models.py

```diff
@@ -1,3 +1,18 @@
 from django.db import models
 
-# Create your models here.
+from mongoengine import EmbeddedDocument, StringField, Document, DateTimeField, ListField, \
+    EmbeddedDocumentField
+
+class User(EmbeddedDocument):
+    user_id: StringField(required=True)
+    username: StringField(required=True)
+
+class Message(EmbeddedDocument):
+    sender = EmbeddedDocumentField(User)
+    message = StringField(required=True)
+
+class Room(Document):
+    id = StringField(primary_key=True)
+    created_at = DateTimeField()
```

Comment:
The field can automatically store the current timestamp when a room is created. You can achieve this by using the default parameter:
```
created_at = DateTimeField(default=datetime.datetime.utcnow)
```